src/useLocalStorage.ts

```diff
@@ -94,6 +104,6 @@ const useLocalStorage = <T>(
   }, [key, setState]);
 
   return [state, set, remove];
```

Comment:
nit: can I suggest we also add the correct return type, so we include the remove function

Reply:
ps I am not involved in the project in any way. I was coincidentally looking to fix the types in my own PR, in my case for the `remove` function . So I thought best to first check no one else was working on this code

Reply:
It looks like it is included already? The return type is a 3-tuple:

```ts
[T, Dispatch<SetStateAction<T>>, () => void]
```

Reply:
sorry you are correct! it is thanks